LibraryApp/src/redux/auth/reducer.js

```diff
@@ -1,41 +1,33 @@
 import immutable from 'seamless-immutable';
+import { createReducer, completeReducer, completeState } from 'redux-recompose';
 
 import { actions } from './actions';
 
-const initialState = {
-  loading: false,
-  logged: false,
-  authError: null
+const stateDescription = {
+  login: null,
+  initialAuth: null
 };
 
-const reducer = (state = immutable(initialState), action) => {
-  switch (action.type) {
-    case actions.INIT_STORED_USER:
-      return state.merge({
-        loading: false,
-        logged: true
-      });
-    case actions.LOGIN_REQUEST:
-      return state.merge({
-        loading: true,
-        authError: null
-      });
-    case actions.LOGIN_SUCCESS:
-      return state.merge({
-        logged: true,
-        loading: false
-      });
-    case actions.LOGIN_FAILURE:
-      return state.merge({
-        logged: false,
-        loading: false,
-        authError: action.payload
-      });
-    case actions.LOGOUT:
-      return state.merge({ ...initialState });
-    default:
-      return state;
+const initialState = completeState(stateDescription);
+
+const reducerDescription = {
+  primaryActions: [actions.LOGIN, actions.AUTH_INIT],
+  override: {
+    [actions.AUTH_INIT_SUCCESS]: (state, action) =>
+      state.merge({
+        initialAuth: action.payload,
+        initialAuthLoading: false,
+        initialAuthError: null,
+        loginLoading: false
+      }),
+    [actions.LOGIN_SUCCESS]: (state, action) =>
```

Comment:
I dont think you need this

Reply:
you was right! i forgot the target attr in the onSuccess() override 😓